test/e2e/tests/setup_e2e_test.go

```diff
@@ -43,7 +43,7 @@ var (
 	minCpuPlatform            = flag.String("min-cpu-platform", "AMD Milan", "Minimum CPU architecture")
 	zones                     = flag.String("zones", "us-east4-a,us-east4-c", "Zones to run tests in. If there are multiple zones, separate each by comma")
 	machineType               = flag.String("machine-type", "n2d-standard-2", "Type of machine to provision instance on")
-	imageURL                  = flag.String("image-url", "projects/ubuntu-os-cloud/global/images/family/ubuntu-minimal-2310-amd64", "OS image url to get image from")
+	imageURL                  = flag.String("image-url", "projects/ubuntu-os-cloud/global/images/family/ubuntu-minimal-2404-lts-amd64", "OS image url to get image from")
```

Comment:
The e2e test was failing with 

```
F0814 21:27:42.152971    8151 setup_e2e_test.go:155] Failed to setup instance gce-pd-csi-e2e-us-east4-a: could not create instance gce-pd-csi-e2e-us-east4-a: API error: googleapi: Error 404: The resource 'projects/ubuntu-os-cloud/global/images/family/ubuntu-minimal-2310-amd64' was not found, notFound
FAIL	sigs.k8s.io/gcp-compute-persistent-disk-csi-driver/test/e2e/tests	60.708s
```

So I updated to one of the images returned from `gcloud compute images list --project=ubuntu-os-cloud | grep ubuntu-minimal`


So I'm trying to bump the image and see if that helps